en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Supported modules
+---
+
+# Commercially supported modules for Silverstripe CMS 5
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to semantic version. They are covered by:
+- our [security release process](./Managing_Security_Issues) and 
+- our [major release policy](./Major_release_policy).
+
+These modules will be supported for the lifetime of Silverstripe CMS 5 in the provided major versions.
+
+## Core Silverstripe CMS modules
+
+These modules provide the core Silverstripe CMS experience. Most Silverstripe CMS projects should install all of them.
+
+Core Module | Supported versions
```

Comment:
```suggestion
Core Module | Supported major release line
```

Reply:
`2` is not a version, it's a line of versions.
This change also applies to the other table below.

If you have a good reason to not make this change, then please use `version` instead of `versions`.